presto-password-authenticators/src/main/java/io/prestosql/plugin/password/filebasedauthenticator/HashingAlgorithm.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.password.filebasedauthenticator;
+
+public enum HashingAlgorithm
+{
+    BCRYPT,
+    PBKDF2
```

Comment:
```
public enum HashingAlgorithm
{
    BCRYPT,
    PBKDF2,
    /**/;
}
```

Reply:
This allows adding new items later without affecting the formatting of existing items

Reply:
Done